backend/runner/proxy/proxy.go

```diff
@@ -96,8 +100,10 @@ func (r *Service) AcquireLease(ctx context.Context, c *connect.BidiStream[ftllea
 
 }
 
-func (r *Service) PublishEvent(ctx context.Context, c *connect.Request[ftldeployment.PublishEventRequest]) (*connect.Response[ftldeployment.PublishEventResponse], error) {
-	event, err := r.controllerDeploymentService.PublishEvent(ctx, connect.NewRequest(c.Msg))
+func (r *Service) PublishEvent(ctx context.Context, c *connect.Request[ftlpubsubv1.PublishEventRequest]) (*connect.Response[ftlpubsubv1.PublishEventResponse], error) {
+	logger := log.FromContext(ctx)
+	logger.Warnf("PUBLISH MESSAGE -------")
```

Comment:
Looks like this is just some debug logs?

Reply:
oops, fixed